src/06_tuples.py

```diff
@@ -36,9 +36,17 @@ def dist(a, b):
 
 # YOUR CODE HERE
 
+# def print_tuple(t):
+#     for ind,val in enumerate(t):
+#         print(val)
+
+def print_tuple(t):
+    for ind,val in enumerate(t):
```

Comment:
since you arent using "ind" within your for loop there is no need to enumerate. this code works so it passes but in this instance you could remove ind and the enumerate() wrapper for cleaner code. 
like this - ``` for val in t: ```

Reply:
lol I way overcomplicated this!